src/__tests__/index.tsx

```diff
@@ -49,10 +49,7 @@ import * as React from 'react'
 export default ({children}) => <div className="sub-dir">{children}</div>
     `.trim(),
     },
-    esbuild: options => {
-      options.external = [...(options.external as Array<string>), 'left-pad']
-      return options
-    },
+    globals: {'left-pad': 'myLeftPad'},
```

Comment:
This is a nice improvement. The whole reason I made it so people could customize rollup was so they could update the external/globals. Having a first-class API for this with `globals` means people won't have to use this hyper-flexible API for this common use-case 👍